.github/workflows/check-versions.yaml

```diff
@@ -32,6 +32,25 @@ jobs:
           ./.github/scripts/update-versions.sh
           git diff
 
+      - name: Setup go
+        uses: actions/setup-go@v4
+
+      - name: Setup crane
+        uses: imjasonh/setup-crane@v0.1
+
+      - name: Update image tags
+        run: |
+          go install github.com/mikefarah/yq/v4@latest
+
+          sudo apt-get install wget apt-transport-https gnupg lsb-release
+          wget -qO - https://aquasecurity.github.io/trivy-repo/deb/public.key | sudo apt-key add -
+          echo deb https://aquasecurity.github.io/trivy-repo/deb $(lsb_release -sc) main | sudo tee -a /etc/apt/sources.list.d/trivy.list
+          sudo apt-get update
+          sudo apt-get install trivy
```

Comment:
There is a trivy action

https://github.com/aquasecurity/trivy-action

Reply:
The action runs trivy itself. We need to script the execution of it. Not sure how to do that with the upstream action.

Reply:
+1 for the trivy action if possible.

Reply:
I looked into this some more. I don't think we can use the action:

1. It doesn't support just the install of trivy. its an action to run trivy.
2. We would no longer be able to just run the script locally on our own machines
3. We'd have to change things to be much much more complicated. It would have to do a dynamic matrix job that then runs all the trivy actions, then somehow tell it to not block on action failures, then run another job gathering up all the results of the trivy actions to do the building of the changes & pr.

So, might technically be possible to do, but feels undesirable.

Reply:
I've been running this script manually since its not merged for a while now and its been very handy to be able to run it outside of gate.